boxoffice/views/admin_order.py

```diff
@@ -18,9 +18,51 @@ def format_assignee(assignee):
             }
 
 
-def format_line_items(line_items):
+def jsonify_admin_orders(data_dict):
+    item_collection_id = data_dict['item_collection'].id
+    order_dicts = []
+    for order in data_dict['orders']:
+        if (order.is_confirmed):
+            order_dicts.append({
+                'invoice_no': order.invoice_no,
+                'id': order.id,
+                'order_date': date_time_format(order.paid_at) if order.paid_at else date_time_format(order.initiated_at),
+                'status': order.is_confirmed,
```

Comment:
`order.is_confirmed` is a bool. So, perhaps makes sense to rename `status` to `confirmed`?